src/components/Tree/Tree.stories.tsx

```diff
@@ -327,3 +328,20 @@ export const WithExpandOnSelect = ({ ...args }: TreeProps) => {
         </div>
     );
 };
+
+export const InsideFlyout = ({ ...args }: TreeProps) => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    return (
+        <div style={{ width: 800 }}>
+            <Flyout
+                trigger={<button type={'button'}>{'Open Tree'}</button>}
```

Comment:
It's fine for me to add this story but can we style a bit so it doesn't look like bare minimum. I would use a Fondue Button here. 
`trigger={<Button>Open Tree</Button>}`